app/controllers/topics.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+helpers do
+  def res(status_code, message)
+    status status_code
```

Comment:
Use def with parentheses when there are parameters. Omit the parentheses when the method doesn't accept any parameters. 
https://github.com/rubocop-hq/ruby-style-guide#method-parens